solr/solrj/src/test/org/apache/solr/client/solrj/embedded/SolrExampleJettyTest.java

```diff
@@ -118,72 +111,22 @@ public void testUtf8PerfDegradation() throws Exception {
     doc.addField("id", "1");
     doc.addField("b_is", IntStream.range(0, 30000).boxed().collect(Collectors.toList()));
 
-    HttpSolrClient client = (HttpSolrClient) getSolrClient();
-    client.add(doc);
-    client.commit();
-    long start = System.nanoTime();
-    QueryResponse rsp = client.query(new SolrQuery("*:*"));
-    System.out.println("time taken : " + ((System.nanoTime() - start)) / (1000 * 1000));
-    assertEquals(1, rsp.getResults().getNumFound());
-  }
-
-  @Ignore
```

Comment:
when was this marked `@Ignore` would be good to find out why?

Reply:
The test was added with the `@Ignore` tag four years ago, so I suspect it wasn't really a test but was to figure some code out....   https://github.com/apache/solr/blame/26195c82493422cb9d6d4bdf9d4452046e7b3f67/solr/solrj/src/test/org/apache/solr/client/solrj/embedded/SolrExampleJettyTest.java#L130.